projects/admin/src/app/circulation/item/item.component.html

```diff
@@ -87,26 +89,49 @@
     </ng-container>
   </div>
   <!-- COLLAPSED DETAILS -->
-  <div class="col-sm-6 mt-2">
-    <ul class="list-unstyled mb-0" id="collapse" [collapse]="isCollapsed" [isAnimated]="true">
-      <li *ngIf="item.loan && item.loan.pickup_location_pid">{{ 'Location' | translate }}:
-        {{ item.loan.pickup_location_pid | getRecord: 'locations' : 'field' : 'code' | async }} {{ item.location.name }}
-      </li>
-      <li *ngIf="item.loan && item.loan.extension_count">{{ 'Renewals' | translate }}: {{ item.loan.extension_count }}
-      </li>
-      <li *ngIf="totalAmountOfFee > 0">{{ 'Fees' | translate }}:
-        {{ totalAmountOfFee | currency: organisation.default_currency }}</li>
+  <div class="col-sm-12 mt-2" *ngIf="!isCollapsed">
+    <dl class="row">
+      <ng-container *ngIf="item.loan && item.loan.pickup_location_pid">
+        <dt class="col-sm-5 col-md-3">{{ 'Location' | translate }}:</dt>
```

Comment:
Why you don't use class `label-title` on element ?
`<dt class="col-sm-5 col-md-3 label-title" translate>Location</dt>`

Reply:
done